posts/panel_release_1.5/index.ipynb

```diff
@@ -0,0 +1,2331 @@
+       "        clearInterval(timer);\n",
+       "        embed_document(root);\n",
+       "      } else if (document.readyState == \"complete\") {\n",
+       "        attempts++;\n",
+       "        if (attempts > 200) {\n",
+       "          clearInterval(timer);\n",
+       "\t  var Bokeh = get_bokeh(root)\n",
+       "\t  if (Bokeh == null || Bokeh.Panel == null) {\n",
+       "            console.warn(\"Panel: ERROR: Unable to run Panel code because Bokeh or Panel library is missing\");\n",
+       "\t  } else {\n",
+       "\t    console.warn(\"Panel: WARNING: Attempting to render but not all required libraries could be resolved.\")\n",
+       "\t    embed_document(root)\n",
+       "\t  }\n",
+       "        }\n",
+       "      }\n",
+       "    }, 25, root)\n",
+       "  }\n",
+       "})(window);</script>"
+      ]
+     },
+     "metadata": {
+      "application/vnd.holoviews_exec.v0+json": {
+       "id": "c67f063c-0d16-4dbe-9f4e-d6a557703f7d"
+      }
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#| echo: false\n",
+    "import datetime as dt\n",
+    "import random\n",
+    "import time\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import panel as pn\n",
+    "import holoviews as hv\n",
+    "\n",
+    "pn.extension('filedropper', inline=False, design='material')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e6c69ede-6b47-42e1-9fb8-7712c211e02b",
+   "metadata": {},
+   "source": [
+    "\n",
+    "<img src=\"./images/collage_1.5.png\" width=\"100%\"></img>\n",
+    "\n",
+    "## What is Panel?\n",
+    "\n",
+    "Panel is an open-source Python library that lets you **easily build powerful tools, dashboards and complex applications entirely in Python**. It has a batteries-included philosophy, putting the PyData ecosystem, powerful data tables and much more at your fingertips. High-level reactive APIs and lower-level callback based APIs ensure you can quickly build exploratory applications, but you aren’t limited if you build complex, multi-page apps with rich interactivity. Panel is a member of the [HoloViz](https://holoviz.org) ecosystem, your gateway into a connected ecosystem of data exploration tools.\n",
+    "\n",
+    "## New release!\n",
+    "\n",
+    "We are very pleased to announce the 1.5.0 release of Panel! While technically a minor release, this minor release hugely expands the scope of what is possible in Panel. A high-level overview of the most important features:\n",
+    "\n",
+    "- Make it trivially easy to build new JS and React based components with hot reload and built-in compilation and bundling\n",
+    "- Add native support for running Panel apps on a FastAPI server\n",
+    "- Add a number of new components including the `Placeholder` pane, `FileDropper` and `TimePicker` widgets, and `ChatStep` component.\n",
+    "- Greatly improved `ChatFeed` UX and support for streaming text chunks to the frontend.\n",
```

Comment:
```suggestion
    "- Greatly improved `ChatFeed` UX and support for streaming text chunks to the frontend.\n",
    "- More flexibility with what is displayed with the `Player` and `DiscretePlayer` widgets"
```

Reply:
I'm happy to mention it but I don't think it quite rises to a topline feature.

Reply:
Yeah a mention would be nice; previously, I felt like Player was too big to fit reasonably in places, or use in the sidebar.

Reply:
https://holoviz-dev.github.io/blog-dev/posts/panel_release_1.5/#player-and-discreteplayer